picard/formats/id3.py

```diff
@@ -112,6 +112,7 @@ class ID3File(File):
         'TCON': 'genre',
         'TMED': 'media',
         'TMOO': 'mood',
+        'TOAL': 'work',
```

Comment:
Work is also used in freetext.

When saving, the work metadata will be saved both as TOAL and TXXX:work.

When loading and TOAL and TXXX:work are both set, I have not tested which takes precedence or whether you get multi-value metadata.